app/api/check-anonymous-data/route.ts

```
@@ -49,8 +60,23 @@ export async function GET(request: NextRequest) {
     });
   } catch (error) {
     console.error("Error checking anonymous data:", error);
+    // Add more detailed error information for debugging
+    const errorMessage = error instanceof Error ? error.message : String(error);
+    const errorStack = error instanceof Error ? error.stack : undefined;
+
+    console.error("Error details:", {
+      message: errorMessage,
+      stack: errorStack,
+      sessionId: sessionId.substring(0, 8) + "...", // Log partial sessionId for debugging
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)